# We don't need to sign the gem and also syntax is not compatible with Ruby 3.2.
From 7534378c4daa4ad8d9fba50a606e97e1beb28ee8 Mon Sep 17 00:00:00 2001
From: Dentrax <furkan.turkal@chainguard.dev>
Date: Tue, 29 Oct 2024 12:32:41 +0300
Subject: [PATCH] remove signing key

Signed-off-by: Dentrax <furkan.turkal@chainguard.dev>
---
 opensearch-ruby.gemspec | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/opensearch-ruby.gemspec b/opensearch-ruby.gemspec
index f86f6980..5013a3b2 100644
--- a/opensearch-ruby.gemspec
+++ b/opensearch-ruby.gemspec
@@ -56,12 +56,6 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files  = %w[README.md USER_GUIDE.md LICENSE.txt]
   s.rdoc_options      = ['--charset=UTF-8']
 
-  signing_key = File.expand_path('gem-private_key.pem')
-  if $PROGRAM_NAME.end_with?('gem') && ARGV.first == 'build' && File.exist?(signing_key)
-    s.signing_key = signing_key
-    s.cert_chain  = ['.github/opensearch-rubygems.pem']
-  end
-
   s.required_ruby_version = '>= 2.5'
 
   s.add_dependency 'faraday', '>= 1.0', '< 3'
-- 
2.39.3 (Apple Git-146)

